src/vm/crossgencompile.cpp

```diff
@@ -384,7 +384,7 @@ extern "C" UINT_PTR STDCALL GetCurrentIP()
     return 0;
 }
 
-void EEPolicy::HandleFatalError(UINT exitCode, UINT_PTR address, LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo, LPCWSTR errorSource)
+void EEPolicy::HandleFatalError(UINT exitCode, UINT_PTR address, LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo, LPCWSTR errorSource, const WCHAR * argExceptionString)
```

Comment:
Nit (I don't actually care if you change it): LPCWSTR is the same as const WCHAR* so this could be written the same as the other strings.

Reply:
Thanks! I think it'd be nice to change it regardless to keep it consistent with the other arguments.. I'll change them when I submit another PR for sending these logs to Windows Event Log.